client/settings/express-checkout/apple-google-pay-item.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl } from '@wordpress/components';
+import interpolateComponents from '@automattic/interpolate-components';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getPaymentMethodSettingsUrl } from '../../utils';
+import ApplePay from 'assets/images/cards/apple-pay.svg?asset';
+import GooglePay from 'assets/images/cards/google-pay.svg?asset';
+import { usePaymentRequestEnabledSettings } from 'wcpay/data';
+import { PaymentRequestEnabledSettingsHook } from './interfaces';
+
+const AppleGooglePayExpressCheckoutItem = (): React.ReactElement => {
+	const [
+		isPaymentRequestEnabled,
+		updateIsPaymentRequestEnabled,
+	] = usePaymentRequestEnabledSettings() as PaymentRequestEnabledSettingsHook;
+
+	return (
+		<li
+			className="express-checkout"
+			id="express-checkouts-apple-google-pay"
+		>
+			<div className="express-checkout__row">
+				<div className="express-checkout__checkbox">
+					<CheckboxControl
+						label={ __(
+							'Apple Pay / Google Pay',
+							'woocommerce-payments'
+						) }
+						checked={ isPaymentRequestEnabled }
+						onChange={ updateIsPaymentRequestEnabled }
+					/>
+				</div>
+				<div className="express-checkout__text-container">
+					<div>
+						<div className="express-checkout__subgroup">
+							<div className="express-checkout__icon">
+								<img src={ ApplePay } alt="Apple Pay" />
+							</div>
+							<div className="express-checkout__label express-checkout__label-mobile">
+								{ __( 'Apple Pay', 'woocommerce-payments' ) }
+							</div>
```

Comment:
Right below, there is another similar div with only difference is the classname `desktop`. 

It might be worth extracting it to a new component with two props "title" and "viewType". With that, it can be reused 8 times across express checkout components here.